kernel/content/ezcontentoperationcollection.php

```diff
@@ -866,6 +870,11 @@ static public function removeNodes( array $removeNodeIdList )
 
             if ( !isset( $objectIdList[$objectId] ) )
                 $objectIdList[$objectId] = eZContentObject::fetch( $objectId );
+            eZAudit::writeAudit( 'location-remove', array( 'Parent Node ID' => $node->attribute( 'parent_node_id' ),
+                                              'Node ID' => $nodeId,
```

Comment:
I am working on patch to show the new main node id in this log but it also good to not that such info is always added to the main_node_update.log regardless.